labelbox/orm/db_object.py

```diff
@@ -244,3 +254,28 @@ def delete(self):
         a call to this you should not use this DB object anymore.
         """
         type(self).bulk_delete([self])
+
+
+def beta(fn):
+
+    def wrapper(self, *args, **kwargs):
+        if not isinstance(self, DbObject):
+            raise TypeError(
+                "Cannot decorate functions that are not functions of `DbOjects` with `beta` decorator"
+            )
+        if not self.client.enable_beta:
+            raise Exception(
+                f"This function {fn.__name__} relies on a beta feature in the api. This means that the interface could change."
+                " Set `enable_beta=True` in the client to enable use of these functions."
+            )
+        execute_fn = self.client.execute
+        try:
+            self.client.execute = partial(execute_fn, beta=True)
+            result = fn(self, *args, **kwargs)
+            if isinstance(result, PaginatedCollection):
+                result.beta = True
```

Comment:
This was so that the PaginatedCollection would keep track of whether or not to call the beta endpoint even when the function decorated with the beta decorator was out of scope. This is no longer used.